Bottom: cf1c326cf6c494cd34338660580bcfb176462429
Top:    fd7f40bfb83878bc362c31332b01602f41a00b85
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-05-01 23:19:16 +0100

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.h b/src/hid/common/object3d.h
index d33551e..0e8b1d5 100644
--- a/src/hid/common/object3d.h
+++ b/src/hid/common/object3d.h
@@ -15,5 +15,5 @@ void object3d_set_appearance (object3d *object, appearance *appear);
 void object3d_add_edge (object3d *object, edge_ref edge);
 void object3d_add_vertex (object3d *object, vertex3d *vertex);
 void object3d_add_face (object3d *object, face3d *face);
-GList *object3d_from_contours (const POLYAREA *contours, double zbot, double ztop, const appearance *master_object_appearance, const appearance *master_topbot_appearance);
+GList *object3d_from_contours (const POLYAREA *contours, double zbot, double ztop, const appearance *master_object_appearance, const appearance *master_top_bot_appearance);
 GList *object3d_from_board_outline (void);
